substrate-node/pallets/pallet-tfgrid/src/migrations/v14.rs

```diff
@@ -9,7 +9,7 @@ use log::{debug, info};
 use sp_std::{marker::PhantomData, vec};
 use tfchain_support::{
     traits::PublicIpModifier,
-    types::{Farm, Node, PublicIP, IP4},
+    types::{Farm, Node, IP4},
 };
 
 #[cfg(feature = "try-runtime")]
```

Comment:
maybe replace `FixPublicIPV13` by `<SomeName>V14` on 2 occurrences below

Reply:
The version is already in the name of the file so I think it's a bit of overhead.